### PR TITLE
feat: Refactor ProposalCard to take secondary actions out of main car…

### PR DESCRIPTION
**Pull request by PaolaSantolloV**
# What does this PR do?

Refactor ProposalCard to take secondary actions out of main card area. I added the CardActions sections.

# Where should the reviewer start?

- app/core/components/EllipsisText/EllipsisText.styles.tsx
- app/core/components/EllipsisText/index.tsx
- app/core/components/ProposalCard/ProposalCard.styles.tsx
- app/core/components/ProposalCard/index.tsx
# How should this be manually tested?

1.- Be on the /projects
 You should see the cards and be able to click on the skills and the tiers links 

# Any background context you want to provide?
The ProposalCard component was changed to solve a browser error regarding rendering tags inside another tag. (changed the href in CardActionArea to an onClickEvent) which removed the ability to click + hold a key to open a new window/tab

# What are the relevant tickets?

https://github.com/wizeline/remix-project-lab/issues/232 [Refactor ProposalCard to take secondary actions out of main card area](https://github.com/wizeline/remix-project-lab/issues/232)
# Screenshots

Before
Screenshot 2023-10-20 at 9 31 16

After
Screenshot 2023-10-19 at 19 40 05

# Questions

# Checklist

- [ ] I added the necessary documentation, if appropriate.
- [ ] I added tests to prove that my fix is effective or my feature works.
- [ ] I reviewed existing Pull Requests before submitting mine.

### CODE REVIEW

**Comment by martinrobled0** [comment]
Can you improve the view when there are many skills?
image

File: app/core/components/ProposalCard/index.tsx [hunk 1]
```diff
@@ -86,75 +86,72 @@ export const ProposalCard = (props: IProps) => {
...
-                        <label
-                          className="ProposalCard__head__description--tier--extra"
-                          title="Maturation framework for innovation projects"
-                        >
-                          <HelpIcon />
+                    <span className="ProposalCard__status--display">
+                      {props.status}
+                    </span>
+                    <div className="ProposalCard__tier">
+                      <Link
```
Comment (martinrobled0) [inline app/core/components/ProposalCard/index.tsx:114]: I checked and the cursor didn’t change to the correct cursor, la manita, can you change it?

**Review by martinrobled0 — CHANGES_REQUESTED** [review]

**Review by CarlosMilano — COMMENTED** [review]
Also noticed Martin issue

**Comment by CarlosMilano** [comment]
> Can you improve the view when there are many skills? image

**Review by AxelUZ — COMMENTED** [review]
I have the same problem that martin mentioned with la manita, in the link to the tiers.

File: app/core/components/ProposalCard/index.tsx (hunk 1 above)
Comment (PaolaSantolloV) [reply app/core/components/ProposalCard/index.tsx:114]: I removed the label and add the cursor as pointer

File: app/core/components/ProposalCard/index.tsx [hunk 2]
```diff
@@ -86,75 +86,70 @@ export const ProposalCard = (props: IProps) => {
...
+              {props.skills &&
+                props.skills[0].name != "" &&
+                props.skills.map((skill) => (
+                  <Chip
+                    key={skill.name}
+                    component="a"
+                    label={skill.name}
+                    sx={{ marginRight: 1, marginBottom: 1 }}
+                    clickable
+                    href={`/projects?discipline=${skill.name}`}
```
Comment (martinrobled0) [inline app/core/components/ProposalCard/index.tsx:102]: here it's _skill_ instead discipline

**Review by martinrobled0 — CHANGES_REQUESTED** [review]

**Review by martinrobled0 — APPROVED** [review]
great!

**Pull request merged**